web/src/submessage.ts

```diff
@@ -29,12 +29,21 @@ const poll_widget_extra_data_schema = z
     })
     .nullable();
 
+export const todo_widget_extra_data_schema = z
+    .object({
+        task_list_title: z.string().optional(),
+    })
+    .nullable();
```

Comment:
Could not understand the intention behind having a `.nullable` here. Now with `todo_widget` having an `extra_data` object/field (ie no longer `null`), I am unable to imagine a case where `extra_data` can be null.
Am I missing something ?

Reply:
All of the `.nullable()`s are for backward compatibility. Though I'm not sure about zforms, polls did not have extra_data from the beginning, and we just introduced it for todos here, so these 2 definitely need to be `nullable`. I don't see any downside in keeping zform `nullable` too, just in case.

Reply:
Makes sense. All LGTM then. Thanks for the clarifications.